Backend/lib/beckn-spec/src/BecknV2/FRFS/Enums.hs

```diff
@@ -74,6 +74,11 @@ data VehicleCategory = METRO | BUS
 
 $(mkHttpInstancesForEnum ''VehicleCategory)
 
+data VehicleVariant = METRO_TRAIN | ASHOK_LEYLAND_AC | MIDI_AC | VOLVO_AC | SPECIAL | MIDI_NON_AC | GSAGAR | EXECUTIVE | ELECTRIC_VEHICLE | ORDINARY
```

Comment:
Better to have EBIX specific things in separate table

Reply:
Done